src/app/layout.js

```diff
@@ -1,28 +1,21 @@
-import localFont from "next/font/local";
-import "./globals.css";
-
-const geistSans = localFont({
-  src: "./fonts/GeistVF.woff",
-  variable: "--font-geist-sans",
-  weight: "100 900",
-});
-const geistMono = localFont({
-  src: "./fonts/GeistMonoVF.woff",
-  variable: "--font-geist-mono",
-  weight: "100 900",
-});
-
-export const metadata = {
-  title: "Sultan Shop",
-  description: "The best shop in the world",
-};
+import Header from "@/components/Header/Header";
+import styles from "./globals.scss";
+import Footer from "@/components/Footer/Footer";
+import { LanguageProvider } from "@inlang/paraglide-next"
+import { languageTag } from "@/paraglide/runtime.js"
```

Comment:
Порядок подлкючения импортов .
```suggestion
import Header from "@/components/Header/Header";
import styles from "./globals.scss";
import Footer from "@/components/Footer/Footer";
import { LanguageProvider } from "@inlang/paraglide-next";
import { languageTag } from "@/paraglide/runtime.js";
```